codebase-pipelines/providers.tf

```
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 1.7"
```

Maybe we should bump this(?). This is quite old now.

---

Yeah looks like 1.7 throughout terraform-platform-modules, 1.8 elsewhere. Not sure what version we are on as a platform, I'll check

---

https://uktrade.atlassian.net/wiki/spaces/DBTP/pages/4340056065/ADR-002+Draft+-+Support+2+minor+releases+of+Terraform

```suggestion
  required_version = "~> 1.8"
```

---

I've updated this but the [CodeBuild terraform test build](https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/763451185160/projects/platform-modules-terraform-tests/build/platform-modules-terraform-tests%3A443ef2ed-1882-477b-b391-2d6e930e6e29/config?region=eu-west-2) is failing because that is on an old version. Do we still need the CodeBuild unit tests if we've moved them to GitHub actions @WillGibson?

---

I'm going to revert this change for now rather than waste time on the almost redundant codebuild image, I've added a squad 1 ticket to fix the versions across the modules https://uktrade.atlassian.net/browse/DBTP-1558